### ISSUE TITLE
dataset settings clarifications

### ISSUE THREAD

**Issue by FabiolaBorrini** [zhengshuchen/HCFNet#1]
Hello!
I want to test the model on my own dataset. 
How should I decide and specify the location where I store the training dataset? 
Moreover, what should be in the mask folder? And how can I provide labels (ground truth) in the training dataset? 

Thank you very much!

**Comment by GanGeeee** [comment]
请问论文结果是在哪个数据集呢？ 我在NUDT-SIRST上复现不出论文结果的80.1的mIOU

**Comment by joan2joan** [comment]
> 请问论文结果是在哪个数据集呢？ 我在NUDT-SIRST上复现不出论文结果的80.1的mIOU

我看论文引用的是DAI那篇 应该是NUAA-SIRST(400多张的那个数据集)

**Comment by GanGeeee** [comment]
> > 请问论文结果是在哪个数据集呢？ 我在NUDT-SIRST上复现不出论文结果的80.1的mIOU
> 
> 我看论文引用的是DAI那篇 应该是NUAA-SIRST(400多张的那个数据集)

可以留个邮箱交流一下吗

**Comment by joan2joan** [comment]
> > > 请问论文结果是在哪个数据集呢？ 我在NUDT-SIRST上复现不出论文结果的80.1的mIOU
> > 
> > 
> > 我看论文引用的是DAI那篇 应该是NUAA-SIRST(400多张的那个数据集)
> 
> 可以留个邮箱交流一下吗

1169862368@qq.com

**Comment by 19451001** [comment]
他这个数据集放在什么位置啊？

**Comment by LouIs184** [comment]
这个数据集样本大小不是512*512，我看代码里面有预处理数据集的代码，但是为什么我把数据集放进去还是会报错

**Comment by zjm-724** [comment]
> 他这个数据集放在什么位置啊？

请问你解决了吗？我也遇到了这个问题

**Comment by loremmoqi** [comment]
> > 他这个数据集放在什么位置啊？
> 
> 请问你解决了吗？我也遇到了这个问题

前辈，请问你解决了吗？我现在也遇到了这个问题，┭┮﹏┭┮

**Comment by loremmoqi** [comment]
> > > 请问论文结果是在哪个数据集呢？ 我在NUDT-SIRST上复现不出论文结果的80.1的mIOU
> > 
> > 
> > 我看论文引用的是DAI那篇 应该是NUAA-SIRST(400多张的那个数据集)
> 
> 可以留个邮箱交流一下吗

你好，请问你解决数据集的问题了吗？可以请教一下你吗
